src/app/components/form/DualCurrencyField/index.tsx

```diff
@@ -9,6 +10,8 @@ export type Props = {
   fiatValue: string;
   label: string;
   hint?: string;
+  amountExceeded?: boolean;
+  maxExceeded?: boolean;
```

Comment:
Since there is also a minimum value for the range it would probably make sense to change this to `rangeExceeded` and also handle amounts < min for consistency?